waiter/test-files/shell-scheduler-backup.json

```diff
@@ -0,0 +1,129 @@
+{
+  "id->service": {
+    "w8r-kitchen": {
```

Comment:
Why "w8r"?

Reply:
I just used it as a name that was short. `w8r` is cute as a name. Renamed to `waiter`.